package.json

```diff
@@ -43,11 +43,11 @@
   },
   "dependencies": {
     "@slack/logger": "^4.0.0",
-    "@slack/oauth": "^3",
-    "@slack/socket-mode": "^2.0.2",
+    "@slack/oauth": "^3.0.2",
+    "@slack/socket-mode": "^2.0.3",
     "@slack/types": "^2.13.0",
-    "@slack/web-api": "^7",
-    "axios": "^1.7.4",
+    "@slack/web-api": "^7.8.0",
```

Comment:
Would this bump make this change a `semver:minor` because new features are introduced? 🤔

I'm thinking we should've had this set to the latest `semver:minor` anyways since features of `@slack/web-api` are exposed from `@slack/bolt` - such as the `assistant` APIs released in [`@slack/web-api@7.5.0`](https://github.com/slackapi/node-slack-sdk/releases/tag/%40slack%2Fweb-api%407.5.0) being required since `@slack/bolt@4.0.0` - but let me know what you think! 🔍